LayoutTests/http/tests/loading/resources/change-location-hash-onload.html

```diff
@@ -0,0 +1,9 @@
+<html>
+<script>
+    window.onload = () => {
+        window.location.hash = Math.random();
+    }
+
+    window.parent.postMessage("hashchange", "*");
```

Comment:
Why do we call this before even setting the location hash?

Reply:
It should be in the `window.onload` after the hash change. Fixed.